src/applications/facility-locator/utils/analytics.js

```diff
@@ -20,42 +20,12 @@ export const recordMarkerEvents = r => {
   }
 };
 
-/**
- * Helper fn to record map zoom and panning events for GA
- */
-export const recordZoomPanEvents = (e, searchCoords, currentZoomLevel) => {
```

Comment:
This utility is not used. DSVA Github search here: https://github.com/search?q=org%3Adepartment-of-veterans-affairs+recordZoomPanEvents&type=code